lib/util/native/platform_check.dart

```diff
@@ -19,7 +19,7 @@ bool checkPlatformIsDesktop() {
 /// This platform supports tray
 /// On linux, this is currently not supported
```

Comment:
I guess this comment is obsolete now

Reply:
The check itself has become redundant as it is the same as `checkPlatformIsDesktop`, but I guess that it gives some extra context to the code that it belongs to, so I don't think it needs to be removed.